Assets/MRTK/Providers/WindowsMixedReality/XR2018/WindowsMixedRealitySpatialMeshObserver.cs

```diff
@@ -232,7 +232,7 @@ public override void Update()
         public override void Resume()
         {
 #if UNITY_WSA
-            if (WaitingForSceneObserverAccess)
+            if (IsRunning)
```

Comment:
Do you mind explaining the change here?

Reply:
this was restoring the existing observer to the original code. it should not have been modified in the scene understanding branch